doc/nrf/ug_thread_tools.rst

```diff
@@ -451,29 +426,68 @@ Building the OpenThread POSIX applications
 Build the OpenThread POSIX applications by performing the following steps:
 
 #. Open a shell in the OpenThread source code directory :file:`ncs/modules/lib/openthread`.
-#. Initialize the build and clean previous artifacts by running the following commands:
+#. Clean previous artifacts by running the following commands::
 
-     .. code-block:: console
+      rm -rf build
 
-        # initialize GNU Autotools
-        ./bootstrap
+#. Build the applications with the required options.
+   For example, to build the ``ot-cli`` application with support for USB transport and Thread v1.1, run the following command::
 
-        # clean previous artifacts
-        make -f src/posix/Makefile-posix clean
+      ./script/cmake-build posix -DOT_SPINEL_RESET_CONNECTION=ON -DOT_THREAD_VERSION=1.1
 
-#. Build the applications with the required options.
-   For example, to build POSIX applications like ``ot-cli`` or ``ot-ncp`` with log output being redirected to the application, run the following command:
+   Alternatively, to build the ``ot-daemon`` and ``ot-ctl`` applications with support for USB transport and Thread v1.1, run the following command::
+
+      ./script/cmake-build posix -DOT_SPINEL_RESET_CONNECTION=ON -DOT_THREAD_VERSION=1.1 -DOT_DAEMON=ON
+
+You can find the generated applications in :file:`./build/posix/src/posix/`.
+
+.. note::
+   The build option ``OT_SPINEL_RESET_CONNECTION`` is required to properly handle USB communication with RCP devices that perform a hard reset.
```

Comment:
You had NCP/RCP here first, but I thought we're using this for RCP only?

Reply:
Yes, OT posix applications are for RCP only.